Spanner/src/SpannerClient.php

```diff
@@ -250,6 +253,15 @@ public function __construct(array $options = [])
         $this->databaseAdminClient = $options['gapicSpannerDatabaseAdminClient']
             ?? new DatabaseAdminClient($clientOptions);
 
+        $channelId = $this->getChannelId($options);
+        // Add the RequestIdHeaderMiddleware to add an identifier to each rpc call made for debugging
+        $requestIdMiddleware = function (callable $handler) use ($channelId) {
```

Comment:
Looks good! I guess with the changes in https://github.com/googleapis/gax-php/pull/640 we can make this `MiddlewareInterface` instead, if we want